src/guide/a11y-basics.md

```diff
@@ -54,12 +54,7 @@ export default {
 </script>
 ```
 
-<p class="codepen" data-height="350" data-theme-id="light" data-default-tab="js,result" data-user="mlama007" data-slug-hash="VwepxJa" style="height: 350px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;" data-pen-title="Skip to Main">
-  <span>See the Pen <a href="https://codepen.io/mlama007/pen/VwepxJa">
-  Skip to Main</a> by Maria (<a href="https://codepen.io/mlama007">@mlama007</a>)
```

Comment:
There are a few examples that used to mention 'Maria' that don't seem to be doing that anymore. Is this intentional?

Reply:
No, just forgot the `name` prop there and I already updated in the recent commits. But in the future I think the core team could migrate everything to the **team/Vue** Codepen account.